ML-Frameworks/pytorch-aarch64/get-source.sh

```diff
@@ -37,11 +39,33 @@ git-shallow-clone https://github.com/pytorch/pytorch.git $PYTORCH_HASH
     apply-github-patch pytorch/pytorch c4c280eb27859221159108356b7c91376202cdd8
 
     # https://github.com/pytorch/pytorch/pull/160184 - Draft: separate reqs for manywheel build and pin
-    apply-github-patch pytorch/pytorch 9a8b0df99eac62e7ec6199dd0223a80d26e2dee0
+    apply-github-patch pytorch/pytorch 6d61f487b6ca98b3d80f9e7ecc0a49a1ab528535
+
+    # https://github.com/pytorch/pytorch/pull/158250 - Ingtegrate INT4→BF16 via KleidiAI, with fallback
+    apply-github-patch pytorch/pytorch 7c55f2af0adf9ce62c2226e739a3c84902fe0048
+    apply-github-patch pytorch/pytorch 8c27947566c85d44bc7dcd7189db5da608453bbb
+    apply-github-patch pytorch/pytorch 15d78c833b032d3c76b70b12a5f2762fa87d2640
+    apply-github-patch pytorch/pytorch 186cbcf641f99a301cb26013e8d74d444ad1dcb9
+    apply-github-patch pytorch/pytorch a6128ce3a0d2080d80e6fa59061d6c085865376c
+    apply-github-patch pytorch/pytorch 52ee4ddc9a5a9cec8793b1ffeb0d74113e3da417
+    apply-github-patch pytorch/pytorch ab2a6760e4a4891accbacb9187cf3782cb4b55c3
+    apply-github-patch pytorch/pytorch 93384233d166dccab5724f9d2e50b6eb3f47cbe6
+
+    # https://github.com/pytorch/pytorch/pull/160080 - VLA Vectorized POC
+    apply-github-patch pytorch/pytorch 506ce2db42df78fd08e48d8af9bcf95d49782dfc
+    apply-github-patch pytorch/pytorch 9ecbee99ab70c3355b9e5956456b7ee7c8b65227
+    apply-github-patch pytorch/pytorch 60ded3ee7c45a28fbd87faad51c9f4d33d272ba7
+    apply-github-patch pytorch/pytorch e873ed21568093a72c04ca68f53112c136c2f417
+    # Optimised SVE exp() implementation
```

Comment:
PR link? And space above for consistency

Reply:
It's the same PR,  I just wanted to call it out.